python/ray/exceptions.py

```diff
@@ -910,6 +907,27 @@ class RayCgraphCapacityExceeded(RaySystemError):
     pass
 
 
+@PublicAPI(stability="alpha")
+class UnpickleableException(RayError):
```

Comment:
Somehow my comment on the previous review got lost, but let's avoid coupling the exception type to the specific implementation of serialization.
```suggestion
@PublicAPI(stability="alpha")
class UnserializableException(RayError):
```

Reply:
technically the issue is during deserialization, but it may be ok to call it UnserializableException. I have made this change